.env.test

```diff
@@ -1,2 +1,4 @@
 REACT_APP_USE_MOCK=true
-REACT_APP_ORCID_BASE_URL=https://sandbox.orcid.org
\ No newline at end of file
+REACT_APP_ORCID_BASE_URL=https://sandbox.orcid.org
+REACT_APP_REDIRECT_SIGN_IN=http://localhost:3000
+REACT_APP_REDIRECT_SIGN_OUT=http://localhost:3000
```

Comment:
.env.test skal sjekkes inn?

Reply:
ja, det er ingen hemmeligheter der, og de variablene der må med for å kunne kjøre testene